### PR TITLE
Update CVE-2018-6667.json

### PR DESCRIPTION
**Pull request by rbyronh**

### CODE REVIEW

File: 2018/6xxx/CVE-2018-6667.json [hunk 1]
```diff
@@ -1,18 +1,86 @@
...
+      }
+   },
+   "data_format": "MITRE",
+   "data_type": "CVE",
+   "data_version": "4.0",
+   "description": {
+      "description_data": [
+         {
+            "lang": "eng",
+            "value": "Authentication Bypass vulnerability in the administrative user interface in McAfee McAfee Web Gateway 7.8.1.0 through 7.8.5 allows remote attackers to execute arbitrary code via Java management extensions (JMX)."
```
Comment (cve-team) [inline 2018/6xxx/CVE-2018-6667.json]: "McAfee" is duplicated in the product name. Is that intentional?

File: 2018/6xxx/CVE-2018-6667.json [hunk 2]
```diff
@@ -1,18 +1,86 @@
...
+                     {
+                        "product_name": "Web Gateway",
+                        "version": {
+                           "version_data": [
+                              {
+                                 "affected": "<",
+                                 "platform": "x86",
+                                 "version_name": "7.8.1",
+                                 "version_value": "7.8.5"
+                              }
```
Comment (cve-team) [inline 2018/6xxx/CVE-2018-6667.json]: I recommend you use two version_data objects to express the range, one with ">=" for "affected" and the other with "<=". And verify the upper bound.

File: 2018/6xxx/CVE-2018-6667.json (hunk 1 above)
Comment (rbyronh) [reply 2018/6xxx/CVE-2018-6667.json]: No, it was accidental. Good catch, thanks.

Comment (cve-team) [reply 2018/6xxx/CVE-2018-6667.json]: So is the upper bound on the version in the JSON correct? SB10241 says that this id involves "7.8.1.0 through 7.8.1.5"

File: 2018/6xxx/CVE-2018-6667.json [hunk 3]
```diff
@@ -1,18 +1,90 @@
 {
```
Comment (cve-team) [inline 2018/6xxx/CVE-2018-6667.json]: Expecting , delimiter: line 22 column 28 (char 715)

File: 2018/6xxx/CVE-2018-6667.json [hunk 4]
```diff
@@ -1,18 +1,91 @@
...
```
Comment (cve-team) [inline 2018/6xxx/CVE-2018-6667.json]: Expecting , delimiter: line 23 column 29 (char 748)

Comment (cve-team) [inline 2018/6xxx/CVE-2018-6667.json]: Expecting , delimiter: line 23 column 29 (char 748)

File: 2018/6xxx/CVE-2018-6667.json [hunk 5]
```diff
@@ -1,18 +1,92 @@
...
```
Comment (cve-team) [inline 2018/6xxx/CVE-2018-6667.json]: Expecting , delimiter: line 24 column 28 (char 779)

**Comment by cve-team** [comment]
Thank you for your submission, expect CVE-2018-6667 to be published on http://cve.mitre.org in the next few hours.

**Pull request merged**